src/convertToObject.js

```diff
@@ -12,7 +13,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const propertiesArray = sourceString
+    .split(';')
+    .map(property => property.split(':'))
+    .filter(property => property.length >= 2)
+    .map(property => [property[0].trim(), property[1].trim()]);
+
+  propertiesArray.map(element => {
+    cssObject[element[0]] = element[1];
+  });
```

Comment:
could you do 1 map instead of 2?

Reply:
I did not succeed. I replace map on forEach, but I don't know is it a good idea :)